src/devtools/packages/devtools-reps/object-inspector/utils/value.tsx

```diff
@@ -216,10 +210,19 @@ export class ValueItem implements IItem {
     await Promise.all(getChildValues(this.contents).map(value => value.loadIfNecessary()));
   }
 
+  recreate() {
+    return new ValueItem({
+      contents: this.contents,
+      isInCurrentPause: this.isInCurrentPause,
+      name: this.name,
+      path: this.path,
+    });
+  }
```

Comment:
This throws me a little. If these properties are now explicitly marked as read-only, and the new `ValueItem` literally has the same values as the exciting one– why do we need to recreate it? Is this a bandaid over something deeper and more serious?

Reply:
These properties are the same, but the `loaded` and `childrenLoaded` properties (which are computed in the item's constructor) may be different because the state of the underlying `ValueFront`s may have changed.